connect-examples/v2/graphql-sample-app/README.md

```diff
@@ -0,0 +1,217 @@
+# Square and GraphQL - Code Example
```

Comment:
Great instructions! I followed them and everything worked as expected without requiring any additional steps 👍

Reply:
nice! Thanks to @mcurtis-squareup for writing an awesome README!